.github/workflows/deploy-docs.yml

```
@@ -1,100 +1,94 @@
+          npm install vitepress@1.6.4
+          npx vitepress build docs
 
-      - name: Rebuild docs container
-        working-directory: ${{ env.DEPLOY_DIR }}
+      - name: Deploy to Cloudflare Pages
+        id: deploy
         env:
-          GIT_COMMIT_SHA: ${{ github.sha }}
+          CLOUDFLARE_API_TOKEN: ${{ secrets.CLOUDFLARE_API_TOKEN }}
+          CLOUDFLARE_ACCOUNT_ID: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
         run: |
```

actionlint (1.7.11)

[error] "github.event.head_commit.message" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)